ChangeLog.d/mbedtls_ecdsa_can_do-unconditional-define.txt

```diff
@@ -0,0 +1,4 @@
+Bugfix
+   * Removes !ECDSA_SIGN_ALT condition around mbedtls_ecdsa_can_do 
+     definition, so that mbedtls_ecdsa_verify_restartable will not 
+     automatically fail.
```

Comment:
This describes the change, but changelog entries are primarily meant for users, so they should explain why the change is needed. Also, please don't abbreviate symbols here: it's not necessarily obvious to readers how to complete them. Also the entry should mention the issue number when there is one. So something like

> Fix a build error when MBEDTLS_ECDSA_SIGN_ALT is defined but not MBEDTLS_ECDSA_VERIFY_ALT. Fixes #7498.

Reply:
Is the updated ChangeLog better?